src/main/java/org/tarantool/jdbc/ds/SQLDataSource.java

```diff
@@ -0,0 +1,163 @@
+package org.tarantool.jdbc.ds;
+
+import org.tarantool.jdbc.SQLConnection;
+import org.tarantool.jdbc.SQLProperty;
+
+import java.io.PrintWriter;
+import java.sql.Connection;
+import java.sql.SQLException;
+import java.sql.SQLFeatureNotSupportedException;
+import java.sql.SQLNonTransientException;
+import java.util.Properties;
+import java.util.concurrent.TimeUnit;
+import java.util.logging.Logger;
+import javax.sql.DataSource;
+
+/**
+ * Simple {@code java.sql.DataSource} implementation.
+ */
+public class SQLDataSource implements TarantoolDataSource, DataSource {
+
+    private PrintWriter logWriter;
+    private String description;
+    private String name;
+
+    private Properties properties = new Properties();
+
+    @Override
+    public Connection getConnection() throws SQLException {
+        return new SQLConnection(makeUrl(), new Properties(properties));
+    }
+
+    @Override
+    public Connection getConnection(String username, String password) throws SQLException {
+        Properties copyProperties = new Properties(properties);
+        SQLProperty.USER.setString(copyProperties, username);
+        SQLProperty.PASSWORD.setString(copyProperties, password);
+        return new SQLConnection(makeUrl(), copyProperties);
+    }
+
+    @Override
+    public PrintWriter getLogWriter() {
+        return logWriter;
+    }
+
+    @Override
+    public void setLogWriter(PrintWriter out) {
+        logWriter = out;
+    }
+
+    @Override
+    public void setLoginTimeout(int seconds) {
```

Comment:
Is it named login timeout rather then init / connection timeout to follow some convention? Init timeout and connection timeout is already two terms for the same thing. Third one makes it even more confusing.

Reply:
`setLoginTimeout` came from DataSource interface. I just adapted this property to the closest 
 (in meaning) property we have already introduced called `initTimeout`.

Reply:
Now I see: https://docs.oracle.com/javase/8/docs/api/javax/sql/CommonDataSource.html , okay.

Reply:
Maybe it worth to name INIT_TIMEOUT as LOGIN_TIMEOUT in PR #172 and have consistent names within JDBC part of the driver?